e2e/test/scenarios/native/native_subquery.cy.spec.js

```diff
@@ -298,8 +296,7 @@ describe("scenarios > question > native subquery", () => {
             "loadQuestion",
           );
 
-          startNewNativeQuestion();
-          SQLFilter.enterParameterizedQuery(`SELECT * FROM {{${tagID}`);
```

Comment:
`enterParameterizedQuery` is doing too much under the hood. No need for it, when we can type directly after we start a new native question.